x/swaprouter/client/query_proto_wrap.go

```diff
@@ -13,9 +18,91 @@ type Querier struct {
 	K swaprouter.Keeper
 }
 
+var sdkIntMaxValue = sdk.NewInt(0)
+
+func init() {
+	maxInt := big.NewInt(2)
+	maxInt = maxInt.Exp(maxInt, big.NewInt(256), nil)
+
+	_sdkIntMaxValue, ok := sdk.NewIntFromString(maxInt.Sub(maxInt, big.NewInt(1)).String())
+	if !ok {
+		panic("Failed to calculate the max value of sdk.Int")
+	}
+
+	sdkIntMaxValue = _sdkIntMaxValue
+}
+
```

Comment:
this is used in tests right? should we move it to tests?

Reply:
This is also used in `EstimateSwapExactAmountOut`